addons/stock/migrations/11.0.1.1/post-migration.py

```diff
@@ -189,6 +195,64 @@ def create_stock_move_line(env):
     )
 
 
+@openupgrade.logging()
+def fill_missing_lots_for_sml(env):
+    """On v10, `reserved_quant_id` field on stock.move.operation.link is
+    filled unless manual operations or extra quantities are added on transfer
+    wizard, which technically means that they fall under this part of code:
+
+    https://github.com/OCA/OpenUpgrade/blob/10.0/addons/stock/models/
+    stock_picking.py#L759
```

Comment:
put something like `on the part of "# 2) then, process the remaining part"`, because the line number may change if the future if we update from upstream again...

Reply:
I can put the permalink